package.json

```diff
@@ -5,30 +5,34 @@
     "npm": "6.x.x"
   },
   "scripts": {
-    "lint": "eslint . --ignore-pattern public --ignore-pattern vendor",
-    "test": "NODE_ENV=test `npm bin`/mocha --require @babel/register --require spec/javascripts/spec_helper.js 'spec/javascripts/**/**_spec.js'"
+    "lint": "eslint . --ignore-pattern public --ignore-pattern vendor --ignore-pattern app/assets/builds",
+    "test": "NODE_ENV=test `npm bin`/mocha --require @babel/register --require spec/javascripts/spec_helper.js 'spec/javascripts/**/**_spec.js'",
+    "build": "webpack"
   },
   "dependencies": {
-    "@rails/webpacker": "^4.3.0",
+    "@rails/ujs": "^6.0.5",
+    "fast-glob": "^3.2.7",
     "identity-style-guide": "^2.2.3",
-    "jquery": "^3.6.0",
-    "jquery-ujs": "^1.2.3",
     "kind-of": "^6.0.3",
     "minimist": "^1.2.6",
     "serialize-javascript": "^3.1.0",
```

Comment:
I don't see that we use these dependencies? I wonder if there was some compat issue in the past with Webpacker where we added them? May be safe to remove now.

Reply:
Let me pull them out and we can see what goes horribly wrong.

Reply:
Nothing went horribly wrong as far as I can tell.